spec/system/navigation_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+RSpec.describe "Main navivation for users to sign in and out" do
+  let(:jobseeker) { create(:jobseeker) }
+  let(:publisher) { create(:publisher) }
```

Comment:
a published should be linked to an organisation, even though it is not relevant for this spec. But if in the future if you need to add anything that is relevant to the organisation, it will not work correctly